core/items/items.py

```diff
@@ -63,8 +63,12 @@ def load():
 						Assets[it.value.texturePath.upper()].value.image, (64, 64))
 					it.value.rect = it.value.image.get_rect()
 				except KeyError:
-					# TODO: Item does not yet have an image
-					pass
+					try:
+						it.value.image = pygame.transform.scale(
+							CustomAssets[it.value.texturePath.upper()].value.image, (64, 64))
```

Comment:
Nee, dit meot zijn: Assets[bla bla bla].value.image. Alle assets zijn custom, het is gwn zo dat we er 1056 hebben in 1 file, en nog een 2e file hebben. Dat betekent niet dat we 2x zoveel memory moeten gebruiken voor 2 classes die letterlijk hetzelfde implementen.

Reply:
done